pollux/lib/sql-doobie/src/main/resources/sql/pollux/V17__add_anoncred_credentials_to_use_columns.sql

```diff
@@ -0,0 +1,4 @@
+-- presentation_records
+ALTER TABLE public.presentation_records
```

Comment:
I'm concerned that we are extending the `presentation_records` table related to the `present_proof` protocol specific to `anoncreds` fields.
I would recommend making a table for these fields and creating the relation from `anoncreds_presentation_records` (or some other name) to generic `presentation_records`.
This approach would enforce the open/close principle and guarantee the isolation of the different concerns.